src/FMI2_md.jl

```diff
@@ -712,12 +712,13 @@ end
 Returns if the FMU model description contains `dependency` information.
 """
 function fmi2DependenciesSupported(md::fmi2ModelDescription)
-    for mv in md.modelVariables
-        if mv.dependencies != nothing && length(mv.dependencies) > 0
-            return true
-        end
-    end 
+    der = md.modelStructure.derivatives
```

Comment:
Please check

Reply:
this should return if `depenendencies` are supported, not only `dependencies.derivatives`